src/os/bluestore/BlueStore.cc

```diff
@@ -12834,6 +12834,14 @@ int BlueStore::_omap_setkeys(TransContext *txc,
 	     << " <- " << key << dendl;
     txc->t->set(prefix, final_key, value);
   }
+
+  string key_tail;
+  bufferlist tail;
+  get_omap_tail(o->onode.nid, &key_tail);
+  if (db->get(prefix, key_tail, &tail) < 0) {
+    txc->t->set(prefix, key_tail, tail);
+  }
+
```

Comment:
this is only needed when we first start using omap... up on line 12817 or so

Reply:
fixed.